client/vaultclient/vaultclient.go

```diff
@@ -358,38 +366,91 @@ func (c *vaultClient) RenewToken(token string, increment int) (<-chan error, err
 		return nil, err
 	}
 
-	// Create a buffered error channel
+	// Create notification channels.
+	renewalCh := make(chan struct{})
 	errCh := make(chan error, 1)
 
 	// Create a renewal request and indicate that the identifier in the
 	// request is a token and not a lease
 	renewalReq := &vaultClientRenewalRequest{
+		renewalCh: renewalCh,
```

Comment:
Hum...good point. I renamed it to `renewalLoopCh` to be  more explicit, but I found very tricky to find a good name 😅 

I feel like `completeCh` and `renewedCh` are a bit misleading because the operation is not complete (the channel will still receive values) and the token may not have been renewed due to an error.

`renewalLoopCh` is kind of _too_ specific because it leaks the notion that the renewal loop exists, but this is an internal struct so probably OK?